f5_sphinx_theme/head.html

```diff
@@ -1,51 +1,18 @@
 <head>
-  
-  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
-  <script type="text"> jQuery.noConflict(true)</script>
-
   {%- block analytics %}
   {# GoogleAnalytics #}
   <!--STAGING:-->
   <script src="https://assets.adobedtm.com/50164577268f481934e5d57dbfb78f929164cd0d/satelliteLib-ea775cfabef5fca8b04e9392f20c986c0724a7c3-staging.js"></script>
   <!--Production:-->
   <script src="https://assets.adobedtm.com/50164577268f481934e5d57dbfb78f929164cd0d/satelliteLib-ea775cfabef5fca8b04e9392f20c986c0724a7c3.js"></script>
   {%- endblock %}
-  
-  {%- block metadata %}
```

Comment:
Can we keep this stuff here and take it out of layout.html? I was trying to make it easy for projects to customize the head without having to replace the whole layout file.

Reply:
I think the reason that I took it out is that it builds 2 head sections (shown below), one from head.html and the other from the basic theme we are extending. I'll put it back in, but I think in the next version we can probably remove the basic theme altogether and add any missing html to the layout. I played around with it and I think there was only a few lines diff by removing the basic theme, but I didn't want to get too carried away with changes in this version. 

![screen shot 2018-04-27 at 4 32 48 pm](https://user-images.githubusercontent.com/12172244/39386494-8125adba-4a3a-11e8-969c-de9a02aece52.png)